README.md

```diff
@@ -8,7 +8,21 @@ This repository contains the source code of several dapp tools
 hand-crafted and maintained by DappHub, along with various Nix
 expressions which are all bundled up into a Nix overlay.
 
-To install tools from this repository follow these instructions:
+Contents:
+
+- [Dapp](./src/dapp)
+- [Seth](./src/seth)
+- [Hevm](./src/hevm)
+- [Ethsign](./src/hevm)
+
+We use `Nix` to for packaging and managing our tools.
```

Comment:
"to for"

Reply:
Also I think this is already mentioned above, where it says

> along with various Nix expressions which are all bundled up into a Nix overlay.

WDYT? Maybe one of these can be removed.